test/github-issues/134/issue-134.ts

```diff
@@ -0,0 +1,31 @@
+import "reflect-metadata";
+import { createTestingConnections, closeTestingConnections, reloadTestingDatabases } from "../../utils/test-utils";
+import { Connection } from "../../../src/connection/Connection";
+import {Post} from "./entity/Post";
+import { expect } from "chai";
+
+describe("github issues > #134 Error TIME is converted to 'HH-mm' instead of 'HH:mm", () => {
+
+    let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        schemaCreate: true,
+        dropSchemaOnConnection: true,
+    }));
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+
+
+    it("should successfully persist the post with creationDate in HH:mm and return persisted entity", () => Promise.all(connections.map(async connection => {
+        const post = new Post();
+        post.title = "Hello Post #1";
+        post.creationDate = new Date();
+        const returnedPost = await connection.entityManager.persist(post);
+
+        expect(returnedPost).not.to.be.empty;
+        returnedPost.should.be.equal(post);
```

Comment:
can you also add few more lines there: load post where post.id = returnedPost.id and assert if its creationDate is correct?